lib/get-config.js

```diff
@@ -107,7 +114,7 @@ function migrateRulesFromRoot(config, source, options) {
       chalk.yellow(
         [
           'Rule configuration has been moved into a `rules` property.',
-          'Please update your `.template-lintrc.js` file.',
+          `Please update your \`${CONFIG_FILE_NAME}\` file.`,
```

Comment:
Mind rewriting this into template strings instead of the `[].join(' ')` thing (which was done to make nicer looking multi-line strings)?

Reply:
@rwjblue this is "autofixed" notation (because our eslint rules)